src/quo2/screens/activity_logs.cljs

```diff
@@ -4,71 +4,126 @@
+                 {:label   "Button 1 type"
+                  :key     :button-1-type
+                  :type    :select
                   :options [{:value "Danger"
-                             :key :danger}
+                             :key   :danger}
                             {:value "Primary"
-                             :key :primary}
+                             :key   :primary}
                             {:value "Success"
-                             :key :success}]}
+                             :key   :success}]}
                  {:label "Button 1 label"
-                  :key :button-1-label
-                  :type :text}
-                 {:label "Button 2 type"
-                  :key :button-2-type
-                  :type :select
+                  :key   :button-1-label
+                  :type  :text}
+                 {:label   "Button 2 type"
+                  :key     :button-2-type
+                  :type    :select
                   :options [{:value "Danger"
-                             :key :danger}
+                             :key   :danger}
                             {:value "Primary"
-                             :key :primary}
+                             :key   :primary}
                             {:value "Success"
-                             :key :success}]}
+                             :key   :success}]}
                  {:label "Button 2 label"
-                  :key :button-2-label
-                  :type :text}
+                  :key   :button-2-label
+                  :type  :text}
                  status-tags/status-tags-options])
 
+(def basic-user-action
+  [[context-tags/group-avatar-tag "Name" {:size :small :color :purple}]
+   "did something here."])
+
+(def complex-user-action
+  [[context-tags/group-avatar-tag "250,000 SNT" {:size :small :color :purple}]
+   "from"
+   [context-tags/group-avatar-tag "Mainnet" {:size :small :color :purple}]
+   "to"
+   [context-tags/group-avatar-tag "Optimism" {:size :small :color :purple}]
+   "on"
+   [context-tags/group-avatar-tag "My savings" {:size :small :color :purple}]])
+
+(def message-with-mention
+  [rn/view {:flex           1
+            :flex-direction :row}
+   [rn/text "Hello"]
+   [rn/text {:style {:color              colors/primary-50
+                     :margin-horizontal  3
+                     :padding-horizontal 3
+                     :background-color   colors/primary-50-opa-10
+                     :border-radius      6}}
+    "@name"]
+   [rn/text "! How are you feeling?"]])
+
 (defn preview []
-  (let [state  (reagent/atom {:title "Activity Title"
-                              :timestamp "Yesterday ∙ 10:41"
-                              :message "Hello Alisher! Do you remember me from the web 3.0 conference in Porto?"
-                              :icon  :placeholder})]
+  (let [state (reagent/atom {:title          "Activity Title"
+                             :timestamp      "Yesterday ∙ 10:41"
```

Comment:
same here, consider changing default message to "Today 00:00" so that designers don't have to alter it check variations